nativelib/src/main/scala/scala/scalanative/unsafe/package.scala

```diff
@@ -144,6 +144,8 @@ package object unsafe extends unsafe.UnsafePackageCompat {
   /** Scala Native unsafe extensions to Arrays */
   implicit class UnsafeRichArray[T](val value: Array[T]) extends AnyVal {
     @inline def at(i: Int): Ptr[T] = value.asInstanceOf[runtime.Array[T]].at(i)
+    @inline def atUnsafe(i: Int): Ptr[T] =
+      value.asInstanceOf[runtime.Array[T]].atUnsafe(i)
   }
```

Comment:
We are already in the `unsafe` package. Actually I would prefer to make `at` directly delegate to the unsafe one because I am never interested in the safe one. But maybe that's too confusing...

Reply:
I can understand we want to make `atUnsafe` as `at` since we are already in `Unsafe` package, but I believe that's too confusing as you mentioned and better to keep as it is 👍 
When we enrich our library using `import scalanative.unsafe._`, people won't see `at` method comes from `unsafe` package at a glance, so naming `atUnsafe` makes sense to me even though `atUnsafe` sounds a bit redundant, my two cents.